packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -40,6 +97,9 @@ export interface IInstanceEngine extends IEngine {
   /** The application used by this engine to perform rotation for a multi-user scenario. */
   readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
 
+  /** Human-readable description of the engine. */
+  readonly description: string;
```

Comment:
There's an argument to be said that making this required is a backwards-incompatible change (if a customer is implementing `IInstanceEngine` themselves).

Perhaps this is better served as a utility function in this module? It seems like the human-readable description can be derived from the other Engine attributes, so I'm not sure we want to force all engines to provide this too.

Reply:
You're absolutely correct, good call. I'll move this to a helper function in `util`.